src/components/spreadsheet/spreadsheet.ts

```diff
@@ -316,6 +317,11 @@ css/* scss */ `
       color: ${TEXT_BODY};
     }
   }
+
+  .o-spreadsheet-topbar-wrapper,
```

Comment:
this commit should come right after the snappy resize, since we already have the introduction of ìsMobile`